src/lib/api/core/base-adapter.ts

```diff
@@ -124,4 +126,17 @@ export class BaseFxDirectiveAdapter extends BaseFxDirective {
   protected _cacheInputString(key = '', source?: string) {
     this._inputMap[key] = source;
   }
+
+  /**
+   * Does this directive have 1 or more responsive keys defined
+   * Note: we exclude the 'baseKey' key (which is NOT considered responsive)
+   */
+  public get usesResponsiveAPI() {
```

Comment:
Something like this might be more concise:

```ts
public get usesResponsiveAPI() {
  const numKeys = Object.keys(this._inputMap).length;
  return !!(numKeys === 1 || !this._inputMap[this._baseKey]);
}
```

Reply:
@crisbeto - that logic is not correct since we are checking for 1 or more non-baseKey values.

```js
public get usesResponsiveAPI() {
  const numKeys = Object.keys(this._inputMap).length;
  return ( !this._inputMap[this._baseKey] && (numKeys > 0));
}
```

Reply:
Fixed.